src/boot/efi/log.c

```diff
@@ -8,6 +8,15 @@
 LogLevel max_log_level = LOG_WARNING;
 static unsigned log_count = 0;
 
+static const int32_t log_colors[] = {
```

Comment:
hmm, SetAttribute() takes an UINTN afaics, hence defining this arrays with a signed type is a bit weird, no?